central/authprovider/service/service_impl.go

```diff
@@ -165,8 +166,13 @@ func (s *serviceImpl) PostAuthProvider(ctx context.Context, request *v1.PostAuth
 	if providerReq.GetLoginUrl() != "" {
 		return nil, errox.InvalidArgs.CausedBy("auth provider loginUrl field is not empty")
 	}
+	if providerReq.GetRequiredAttributes() != nil {
```

Comment:
I think it's fine to allow setting this field via API. The only problem I see is that users might wonder what this field does and request documentation on it...

Reply:
I agree with the sentiment that that may be the case, but on the top of my head couldn't come up with a different, non-hackish solution to allow this specific setting to be set with auth provider.
If we had separate proto definitions for API presentation and storage, we could handle this better I feel like.

Reply:
What I'm saying is that let's mark this required attributes feature experimental and roll it out _normally_, meaning, everyone can use this feature via API.

Reply:
I'm fine with making this available during creation via API.
I'm wondering whether a changelog entry would also be in store - but we can also add it later.